vectorbendertransformers.py

```diff
@@ -175,8 +166,8 @@ def __init__(self, pairsLayer, restrictToSelection):
 
     def map(self, p):
 
-        return QgsPoint( self.a*p.x()+self.b*p.y()+self.c, self.d*p.x()+self.e*p.y()+self.f )
-
+        return QgsPointXY( self.a*p.x()+self.b*p.y()+self.c, self.d*p.x()+self.e*p.y()+self.f ) #Almerio: era QgsPoint
```

Comment:
Remove these unnecessary `#Almerio: era QgsPoint` comments  (for the whole file)

Reply:
Done!